model.py

```diff
@@ -0,0 +1,44 @@
+class CNN(nn.Module):
```

Comment:
Have you tested that both of these networks work (i.e. they learn, and have reasonable accuracy on the test set)?

Reply:
First model has accuracy 37%, second 36%